modules/op-resolver-git-anon.adoc

```diff
@@ -0,0 +1,124 @@
+// This module is included in the following assembly:
+//
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="resolver-git-anon-specify_{context}"]
+= Specifying a remote pipeline or task using the Git resolver for anonymous cloning
+
+When creating a pipeline run, you can specify a remote pipeline from a Git repository using anonymous cloning. When creating a pipeline or a task run, you can specify a remote task from a Git repository.
+
+.Procedure
+
+. To specify a remote pipeline or task from a Git repository, use the following reference format in the `pipelineRef` or `taskRef` spec:
++
+[source,yaml]
+----
+# ...
+  resolver: git
+  params:
+  - name: url
+    value: <git_repository_url>
+  - name: revision
+    value: <branch_name>
+  - name: pathInRepo
+    value: <path_in_repository>
+# ...
+----
++
+.Supported parameters for the Git resolver
+|===
+| Parameter | Description | Example value
+
+| `url`
+| The URL of the repository, when using anonymous cloning.
+| `+https://github.com/tektoncd/catalog.git+`
+
+| `revision`
+| The Git revision in the repository. You can specify a branch name, a tag  name, or a commit SHA hash.
+| `aeb957601cf41c012be462827053a21a420befca` +
+`main` +
+`v0.38.2`
+
+| `pathInRepo`
+| The path name of the YAML file in the repository.
+| `task/golang-build/0.3/golang-build.yaml`
+|===
++
+[NOTE]
+====
+To clone and fetch the repository anonymously, use the `url` parameter. Do not specify the `url` parameter and the `repo` parameter together.
+====
++
+If the pipeline or task requires additional parameters, provide these parameters in `params`.
+
+The following example pipeline run references a remote pipeline from a Git repository:
+
+[source,yaml]
+----
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: git-pipeline-reference-demo
+spec:
+  pipelineRef:
+    resolver: git
+    params:
+      - name: url
+        value: https://github.com/tektoncd/catalog.git
+      - name: revision
+        value: main
+      - name: pathInRepo
+        value: pipeline/simple/0.1/simple.yaml
+      - name: sample-pipeline-parameter
+        value: test
```

Comment:
Why are we putting this "random" parameter here ? It might render the request invalid.